spec/helpers.lua

```diff
@@ -763,6 +768,22 @@ local function kong_exec(cmd, env)
   cmd = cmd or ""
   env = env or {}
 
+  -- Insert the Lua path to the custom-plugin fixtures
+  local lp
+  for k,v in pairs(env) do
+    if k:upper() == "LUA_PACKAGE_PATH" then
+      lp = k
+      break
+    end
+  end
```

Comment:
I think we can get rid of such a loop and the unnecessary cognitive load it requires and simply perform a lookup with both lower-case and upper-case keys. Upper-case is not supported anyways, so lower-case is even good enough. Having the `CUSTOM_PLUGIN_PATH` first would also be preferable.

```lua
if not env.lua_package_path then
  env.lua_package_path = CUSTOM_PLUGIN_PATH

else
  env.lua_package_path = CUSTOM_PLUGIN_PATH .. ";" .. env.lua_package_path  
end
```